### PR TITLE
Add mullvad module

### LINKED ISSUE #187: mullvad: add option

### PR DESCRIPTION
**Pull request by CrazyStevenz**
Closes #187

### CODE REVIEW

**Comment by CrazyStevenz** [comment]
Closes #187

File: system/applications/modules/mullvad.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  ...
+}:
+
+let
+  inherit (lib) mkIf optional;
+
+  cfg = config.icedos.applications;
+in
+mkIf (cfg.mullvad.enable) {
+  environment.systemPackages = [ ] ++ optional (cfg.mullvad.gui) pkgs.mullvad-vpn;
```
Comment (IceDBorn) [inline system/applications/modules/mullvad.nix:14]: There's no need for optional here. Just use mkIf. 
```nix
environment.systemPackages = mkIf (cfg.mullvad.gui) [ pkgs.mullvad-vpn ];
```

File: config.toml [hunk 2]
```diff
@@ -62,6 +62,10 @@ sites = [
   "https://web.skype.com/",
 ]
 
+[icedos.applications.mullvad]
+enable = true
```
Comment (IceDBorn) [inline config.toml:66]: Default it to false.

**Review by IceDBorn — CHANGES_REQUESTED** [review]

File: system/applications/modules/mullvad.nix (hunk 1 above)
Comment (CrazyStevenz) [reply system/applications/modules/mullvad.nix:14]: Yep, was testing something else and forgot to change it back, fixed.

File: config.toml (hunk 2 above)
Comment (CrazyStevenz) [reply config.toml:66]: Done

**Review by IceDBorn — APPROVED** [review]

**Pull request merged**